stable/zetcd/requirements.lock

```diff
@@ -1,11 +1,6 @@
 dependencies:
-- alias: ""
-  condition: ""
-  enabled: false
```

Comment:
Note that my script doesn't yet take account of these options. I also don't yet see documentation of these. Is there some place besides https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/requirements.md#conditions-and-tags I should look?

Reply:
I'm assuming this is correct, since the lock file is updated with `helm dependency update`.